configure.ac

```diff
@@ -5353,11 +5353,18 @@ AC_CONFIG_FILES([Makefile \
 AC_OUTPUT
 
 if test ${device_name} = "ch4" ; then
+    t_netmod=$ch4_netmods
+    if test "$ch4_netmods" = "ofi" -a "$with_libfabric" = "embedded"; then
+        t_netmod="ofi (using embedded libfabric)"
+    elif test "$ch4_netmods" = "ucx" -a "$with_libfabric" = "embedded"; then
+        t_netmod="ucx (embedded)"
```

Comment:
should be checking `$with_ucx` in the conditional

Reply:
Thanks. Fixed.